src/main/java/edu/ksu/canvas/attendance/controller/ExceptionController.java

```diff
@@ -1,8 +1,10 @@
 package edu.ksu.canvas.attendance.controller;
 
+import edu.ksu.canvas.attendance.exception.AttendanceAssignmentException;
 import edu.ksu.canvas.exception.InvalidOauthTokenException;
 import edu.ksu.canvas.attendance.exception.MissingSisIdException;
 import edu.ksu.lti.launch.exception.OauthTokenRequiredException;
+import org.springframework.beans.factory.annotation.Autowired;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.springframework.beans.factory.annotation.Autowired'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AUselessImportCheck)